### DISCUSSION TITLE
Error message enhancements for prerendering / fallback config issues

### DISCUSSION THREAD

**Discussion by dwabyick** [vercel/next.js#37089] [Ideas]
### Describe the feature you'd like to request

I was going through the nextjs-blog example, and hit an issue where I set 'fallback:true' to see what it would be do. No errors on dev, but building for prod gave me only a partially-helpful error. 

Error occurred prerendering page "/posts/[id]". Read more: https://nextjs.org/docs/messages/prerender-error
TypeError: Cannot read properties of undefined (reading 'title')
    at Post (/Users/dwabyick/localcode/nextjs-blog/.next/server/pages/posts/[id].js:41:40)

### Describe the solution you'd like

The link to the 'read more' seemed to anticipate the 'fallback' problem so that's helpful,  but ideally it would say something to indicate this was a problem with 'fallback'

```
Error occurred prerendering fallback routes of page "/posts/[id]".  Read more about fallback config here: <fallback link> 
TypeError: Cannot read properties of undefined (reading 'title')
```

### Describe alternatives you've considered

None. Not a critical issue. Just a tiny enhancement.

**Comment by ijjk** [comment]
Hi, this sounds like a good error message to show when prerendering a fallback, would definitely accept a PR for this if someone is interested in contributing this!
